views/tutors.html

```diff
@@ -30,6 +32,36 @@ <h3 class="d-inline" id="year-current" data-year='{{ @year }}'>{{ @year }} - {{@
         <check if="{{ @currentYear != @year }}">
             <button id="current-year" class="btn btn-info mt-3">Set Current Year</button>
         </check>
+        <br>
+
+        <!-- Filter buttons -->
+        <div class="mt-3">
+            <input type="hidden" value="{{ @status }}" id="status">
+            <check if="{{ @status == 'incomplete' }}">
+                <true>
+                    <button class="btn btn-primary filter">Incomplete</button>
+                </true>
+                <false>
+                    <a href="/tutors/{{ @year }}&incomplete"><button class="btn btn-secondary filter">Incomplete</button></a>
+                </false>
+            </check>
+            <check if="{{ @status == 'all' }}">
+                <true>
+                    <button class="btn btn-primary filter">All</button>
+                </true>
+                <false>
+                    <a href="/tutors/{{ @year }}&all"><button class="btn btn-secondary filter">All</button></a>
+                </false>
+            </check>
+            <check if="{{ @status == 'complete' }}">
+                <true>
+                    <button class="btn btn-primary filter">Complete</button>
+                </true>
+                <false>
+                    <a href="/tutors/{{ @year }}&complete"><button class="btn btn-secondary filter">Complete</button></a>
```

Comment:
(Not a Correction) correct me if I'm wrong but from my understanding you are passing in a variable through the link and checking what she wants it to be filtered by. After you figure this out you choose from the three queries based on the variable. Awesome job man!